tests/contrib/dbapi_async/test_dbapi_async.py

```diff
@@ -95,6 +98,36 @@ async def test_fetchone_wrapped_is_called_and_returned(self):
         assert "__result__" == await traced_cursor.fetchone("arg_1", kwarg1="kwarg1")
         cursor.fetchone.assert_called_once_with("arg_1", kwarg1="kwarg1")
 
+    @mark_asyncio
+    async def test_cursor_async_connection(self):
+        """Checks whether connection can execute operations with async iteration."""
+
+        cursor = self.cursor
+        traced_cursor = TracedAsyncCursor(cursor, Pin("dbapi_service", tracer=self.tracer), {})
+        async with traced_cursor as cursor:
+            await cursor.execute("DROP TABLE IF EXISTS {}".format(TEST_TABLE))
+            await cursor.execute(
+                """CREATE TABLE {} (
+                        a INT,
+                        b VARCHAR(32)
+                        )
+                        """.format(
+                    TEST_TABLE
+                )
+            )
```

Comment:
Do we need to create a table for this test? Can we just execute a `select` query. It would be nice if this test did not require clean up.